File: Persona/alumno.py
from persona import Persona

class Alumno(Persona):
    
    def __init__(self, nombre, apellido, matricula, plan):
        super().__init__(nombre, apellido)
        self.__matricula = matricula
        self.__plan = plan
        
        
    def __str__(self):
        return super().__str__() + ", " + str(self.__matricula) + ", " + str(self.__plan)
    
    
    def get_matricula(self):
        return self.__matricula
    
    
    def get_plan(self):
        return self.__plan

File: Persona/Alumno.java
public class Alumno extends Persona {

    private int matricula;
    private String plan;


    public Alumno(String nombre, String apellido, int matricula, String plan) {
        super(nombre, apellido);
        this.matricula = matricula;
        this.plan = plan;
    }


    public int getMatricula() {
        return matricula;
    }
    
    
    public String getPlan() {
        return plan;
    }


    public String toString() {
        return (super.toString() + ", " + Integer.toString(matricula) + ", " + plan);
    }

}
